src/static/js/modules/breakpoint-handler.js

```diff
@@ -131,3 +131,9 @@ MobileOnlyExpandable.prototype.openExpandable = function () {
         this.expandableTarget.click();
     }
 };
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bebb217d926085fc87d071fa3cb51ac548f9dc34